src/overlays/actors/ovl_kaleido_scope/z_kaleido_debug.c

```diff
@@ -1,5 +1,647 @@
+                        }
+                        if (CHECK_BTN_ALL(input->press.button, BTN_CUP)) {
+                            gSaveContext.inventory.equipment ^= (8 << gEquipShifts[i]);
+                        }
+                    }
+                } else if (curSection < 0x44) {
+                    i = curSection - 0x38;
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                        gSaveContext.inventory.dungeonItems[i] ^= 4;
+                    }
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN)) {
+                        gSaveContext.inventory.dungeonItems[i] ^= 2;
+                    }
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                        gSaveContext.inventory.dungeonItems[i] ^= 1;
+                    }
+                } else if (curSection == 0x5B) {
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CUP) || CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                        gSaveContext.inventory.gsTokens++;
+                    } else if (CHECK_BTN_ALL(input->press.button, BTN_CDOWN) ||
+                               CHECK_BTN_ALL(input->press.button, BTN_CRIGHT)) {
+                        gSaveContext.inventory.gsTokens--;
+                        if (gSaveContext.inventory.gsTokens <= 0) {
+                            gSaveContext.inventory.gsTokens = 0;
+                        }
+                    }
+                } else if (curSection < 0x5C) {
+                    i = curSection - 0x44;
+                    if (CHECK_BTN_ALL(input->press.button, BTN_CUP) || CHECK_BTN_ALL(input->press.button, BTN_CLEFT)) {
+                        gSaveContext.inventory.questItems ^= gBitFlags[i];
+                    }
+                }
+            }
+            break;
+    }
+
+    // Draws an highlight on the selected section
+    gDPPipeSync(POLY_OPA_DISP++);
+    gDPSetRenderMode(POLY_OPA_DISP++, G_RM_XLU_SURF, G_RM_XLU_SURF2);
+    gDPSetCombineMode(POLY_OPA_DISP++, G_CC_PRIMITIVE, G_CC_PRIMITIVE);
+    gDPSetPrimColor(POLY_OPA_DISP++, 0, 0, 0, 0, 200, 120);
+
+    if (curSection == 0) {
+        gDPFillRectangle(POLY_OPA_DISP++, sSectionPositions[curSection][0], sSectionPositions[curSection][1],
+                         sSectionPositions[curSection][0] + 45, sSectionPositions[curSection][1] + 16);
+    } else if ((curSection >= 0x1B) || (curSection == 0x5B)) {
+        gDPFillRectangle(POLY_OPA_DISP++, sSectionPositions[curSection][0] - 2, sSectionPositions[curSection][1],
+                         sSectionPositions[curSection][0] + 14, sSectionPositions[curSection][1] + 16);
+    } else {
+        gDPFillRectangle(POLY_OPA_DISP++, sSectionPositions[curSection][0] - 4, sSectionPositions[curSection][1],
+                         sSectionPositions[curSection][0] + 24, sSectionPositions[curSection][1] + 16);
+    }
+
+    // Handles exiting the inventory editor with the L trigger
```

Comment:
imagine calling the L button a trigger peepoMad

Reply:
It's a trigger tho peepoCry